hydro_deploy/core/src/localhost/mod.rs

```diff
@@ -163,7 +163,10 @@ impl LaunchedHost for LaunchedLocalhost {
                 "dwarf,64000",
                 "-o",
             ])
-            .arg(&perf.output_file)
+            .arg(
+                perf.perf_outfile
+                    .expect("perf_outfile should be set for localhost perf."),
```

Comment:
My guess is this will be changed later when we integrate inferno support for localhost?

Reply:
Yup